freqtrade/exchange/bitget.py

```diff
@@ -27,11 +27,11 @@ class Bitget(Exchange):
     _ft_has: FtHas = {
         "order_time_in_force": ["GTC", "FOK", "IOC"],
         "trades_has_history": True,
+        "funding_fee_candle_limit": 100,
```

Comment:
This should stay in the "futures" branch.
funding fees are irrelevant for spot trading.

If this caused a problem in tests (i suspect that's why you moved it) - then your test setup is wrong.